packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -20,13 +20,12 @@
 							>
 							<Button
 								icon="pi pi-cog"
-								label="Extract Configurations"
+								label="Extract configurations from inputs"
```

Comment:
Per @ngraham76 designs
```suggestion
								label="Extract configurations from connected ressources"
```

Reply:
Neil said he liked this better `Extract configurations from inputs` , hes revamping the designs